csharp/IceStorm/Weather/Sensor/Program.cs

```diff
@@ -12,7 +12,7 @@
 // Create a proxy to the IceStorm topic manager.
 IceStorm.TopicManagerPrx topicManager = IceStorm.TopicManagerPrxHelper.createProxy(
     communicator,
-    "ClearSky/TopicManager:tcp -p 4061");
+    "ClearSky/TopicManager:tcp -p 4061 -h localhost");
```

Comment:
We should not showcase host-free direct proxies.

Reply:
We should update the other demos as well.